dashboard.el

```diff
@@ -254,5 +261,9 @@ If MESSAGEBUF is not nil then MSG is also written in message buffer."
    (redisplay))
   (add-hook 'after-init-hook '(lambda () (switch-to-buffer "*dashboard*"))))
 
+;; Forward declartions for optoinal dependency to keep check-declare happy.
+(declare-function projectile-load-known-projects "projectile.el:ext")
+(declare-function projectile-relevant-known-projects "projectile.el:ext")
```

Comment:
I think it should be the case given this:
https://github.com/jwiegley/emacs-release/blob/master/lisp/emacs-lisp/check-declare.el#L52